lib/hsa/mcwamp_hsa.cpp

```diff
@@ -2167,6 +2175,9 @@ class HSADevice final : public KalmarDevice
             throw Kalmar::runtime_exception("HCC_CHECK_COPY can only be used on machines where accelerator memory is visible to CPU (ie large-bar systems)", 0);
         }
 
+        g_agentToDeviceMap.insert(std::pair<uint64_t, HSADevice*> (agent.handle, this));
```

Comment:
`g_agentToDeviceMap` is populated here, and would be queried by `HSADevice::get_seqnum_from_agent()`. But in the PR there is actually no one calling that static member function, thus making this map unused.